webpack.config.js

```diff
@@ -56,56 +56,40 @@ const htmlRule = {
   ],
 };
 
-// We compile our own code with ts-loader
-const typescriptRule = {
-  test: /\.tsx?$/,
-  use: [
-    {
-      loader: "ts-loader",
-      options: {
-        transpileOnly: true,
-      },
-    },
-  ],
-  exclude: /node_modules/,
-};
-
-// Third party modules are compiled with babel since using ts-loader that much causes webpack to run out of memory
-const ModulesRule = {
-  test: /\.js$/,
-  use: [
-    {
-      loader: "babel-loader",
-      options: {
-        cacheDirectory: ".cache/babel",
-        presets: [["@babel/preset-env", { targets: { ie: "11" }, useBuiltIns: false }]],
-      },
-    },
-  ],
-  include: /node_modules/,
-  // Exclude large modules we know don't need transpiling
-  exclude: /vega|monaco|plotly/,
-};
-
 module.exports = function (env = {}, argv = {}) {
   const mode = argv.mode || "development";
-  const rules = [fontRule, lessRule, imagesRule, cssRule, htmlRule, typescriptRule];
+  const target = mode === "development" ? "es2018" : "es2017";
```

Comment:
Curious, why?

Reply:
Good Q. I am mostly going off this thing that Google put out that says ES2017 is the safe bet for "modern" https://web.dev/publish-modern-javascript/. But ES2018 does have some nice things like `for await` and in dev mode its nice to use see the actual code rather than transpiled code. So... does it matter? Prob not. ES2017 everywhere would be fine.